source/administration/changelog.md

```diff
@@ -4,6 +4,121 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
+ - See [documentation](https://docs.gitlab.com/omnibus/gitlab-mattermost/#upgrading-gitlab-mattermost-from-versions-prior-to-11-0) to learn more.
+
+### Improvements
+
+#### Web User Interface
+ - Added a setting that automatically answers messages sent to a user who is away.
+ - Added a loader for channel invite modal.
+ - Made Shift+up switch keyboard focus to RHS if it's already open to the current thread.
+ - Removed WebRTC end user setting.
+ - Removed support for transparent team icons and added ability to remove team icons.
+ - Added an on-hover effect for image link previews.
+
+#### Performance
+ - Added structured logging.
+ 
+ #### Plugins
+ - Added better plugin error handling and reporting.
+ 
+ #### Slash Commands
+ - Added ``/invite`` slash command to invite users to a channel.
+ - Added ``/platform`` list of commands to teams command in CLI.
+ - Improved slash command error message when payload is invalid JSON.
+ 
+ #### Other (XXXX/// Need help with a category)
+ - Removed ``mm_config`` and ``mm_license`` global state from webapp.
+ - Added support to REST API for sending ephemeral messages to users.
+ - Investigated avoiding duplicating unique indexes.
+ - Added a CLI Command to "Reset to Default" for permission system.
+ - Added a CLI Platform command to change user email address.
+ - Investigated "Download" link to external resources may not actually trigger download.
+ - Investigated switching back to npm.
+ - Upgraded minio-go library to v6.0.0.
+
+### Bug Fixes
+
+ - Fixed an issue where sign-up and team joining were broken on Mattermost server.
+ - Fixed an issue with a mention badge missing for DMs in CTRL/CMD+K channel switcher.
+ - Fixed an issue with weird spacing in post control menu of a post by another user.
+ - Fixed an issue where using CMD/CTRL+K to switch to DM failed with errors in console.
+ - Fixed an issue where focus with CTRL/CMD+SHIFT+L was always set to the right-hand side when reply thread was open.
+ - Fixed an issue where a user added to a channel wasn't immediately removed from other users' add-members list.
+ - Fixed an issue where 'Copy Link' context menu option was partially hidden when right-clicking a team in team sidebar.
+ - Fixed an issue where a channel name overlapped the star icon in channel header.
+ - Fixed an issue where the mention badge in team sidebar was cut off by the team icon border.
+ - Fixed an issue where there was no on-hover effect for buttons in APIv3 deprecation system notice, and in System Console and Integrations pages.
+ - Fixed an issue where ``[...]`` menu was missing on other users' posts if user was not an admin.
+ - Fixed an issue where authdata failed to migrate properly during migration from LDAP to SAML.
+ - Fixed an issue where username in "is typing" message was missing.
+ - Fixed an issue where a message time displayed the long timestamp instead of the short form.
+
+### Compatibility
+   
+#### Upcoming Deprecated Features in Mattermost v5.0
```

Comment:
There are also upgrade notes added for the 4.10 release branch https://github.com/mattermost/docs/blob/v4.10-documentation/source/administration/important-upgrade-notes.rst